test/common/http/http1/codec_impl_test.cc

```diff
@@ -894,15 +921,23 @@ TEST_F(Http1ServerConnectionImplTest, Http11AbsoluteEnabledNoOp) {
   expectHeadersTest(Protocol::Http11, true, buffer, expected_headers);
 }
 
-TEST_F(Http1ServerConnectionImplTest, Http11InvalidRequest) {
+TEST_P(Http1ServerConnectionImplTest, Http11InvalidRequest) {
+  if (parser_impl_ == ParserImpl::BalsaParser) {
```

Comment:
Is there a plan to resolve behavioral differences?
For example for HTTP/3 we had a macro which did this and a TODO at the macro definition to remove it after resolving all of the behavioral differences.

Reply:
Yes, the plan is to remove all differences.  I'm adding a TODO to each of the disabled tests for clarity.